src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/ndmis/performance/ReferralsProcessor.kt

```diff
@@ -36,7 +36,7 @@ class ReferralsProcessor(
       eosrSubmittedAt = referral.endOfServiceReport?.submittedAt?.let { t -> NdmisDateTime(t) },
       endReasonCode = referral.endRequestedReason?.code,
       endReasonDescription = referral.endRequestedReason?.description,
-      concludedAt = referral.concludedAt?.let { t -> NdmisDateTime(t) },
+      concludedAt = referral.concludedAt?.let { t -> NdmisDateTime(t) }
```

Comment:
A minor one, we may have to include the trailing comma as part of the new config